pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -428,6 +428,15 @@ func (s *Server) makeKubeConfigController(args *PilotArgs) (model.ConfigStoreCac
 	return c, nil
 }
 
+func (s *Server) kubeConfigStore() model.ConfigStore {
+	for _, cs := range s.ConfigStores {
```

Comment:
just curious, why the preference to the first occurrence of `s.ConfigStores` instead of just `s.configController`? asking because i really dont know what the difference is.

Reply:
The aggregate controller isn't set up to support writes. I may end up just doing `aggController = MakeWritableCache(ConfigStores, k8sStore or memoryStore)` but rn I'm just getting the core logic working.